### PR TITLE
translate changeset types

### PR DESCRIPTION
**Pull request by mcbuddha**

### CODE REVIEW

**Comment by thenonameguy** [comment]
LGTM

File: src/hu/ssh/github_changelog/formatters/markdown.clj [hunk 1]
```diff
@@ -5,6 +5,15 @@
...
     [schema.core :as s]))
 
+(def type-name-map
+  {:fix "Bug Fixes"
+   :chore "Chores"
+   :feat "Features"})
+
+(s/defn translate-type :- s/Str
+  [type :- s/Str]
+  (or ((keyword type) type-name-map) type))
```
Comment (raszi) [inline src/hu/ssh/github_changelog/formatters/markdown.clj]:
```clojure
(get type-name-map (keyword type) type)
```

Comment (thenonameguy) [inline src/hu/ssh/github_changelog/formatters/markdown.clj]: Actually:
```clojure
((keyword type) type-name-map type)
```

File: src/hu/ssh/github_changelog/formatters/markdown.clj [hunk 2]
```diff
@@ -5,17 +5,29 @@
...
+  (get type-name-map (keyword type) type))
+
 (s/defn format-change :- s/Str
   [change :- Change]
   (str (markdown/emphasis (:scope change))
        " "
        (:subject change)
-       (if-let [issues (:issues change)]
-         (str ", closes " (str-map (partial apply markdown/link) issues)))))
+       (let [issues (:issues change)]
```
Comment (raszi) [inline src/hu/ssh/github_changelog/formatters/markdown.clj]: I think the proposed solution with

```clojure
(seq (:issues change))
```

is way shorter.

Comment (thenonameguy) [inline src/hu/ssh/github_changelog/formatters/markdown.clj]: Agreed.

**Comment by raszi** [comment]
Please merge master

**Pull request closed without merging**